src/passes/LogExecution.cpp

```diff
@@ -39,6 +41,8 @@ namespace wasm {
 Name LOGGER("log_execution");
 
 struct LogExecution : public WalkerPass<PostWalker<LogExecution>> {
+  std::unordered_map<Function*, Index> functionOrdinals;
```

Comment:
What is a "function ordinal"? I don't see "ordinal" anywhere existing in `src/`.

If it is a new concept please add a comment explaining it.

If this is a function index, please rename it to that. Skimming the code below that might be the case?

Reply:
They are the function indices. Good idea, I'll rename.

I'll post this in a bit, I find there is some kind of bug in my use, and not sure if it could be related to this code path; so I'll investigate that and then update this PR.